lib/web/scripts/node-join/install.sh

```diff
@@ -610,7 +610,7 @@ else
 fi
 
 # use OSTYPE variable to figure out host type/arch
-if [[ "${OSTYPE}" == "linux-gnu"* ]]; then
```

Comment:
Is there a reason we're using $OSTYPE rather than checking against `uname` ? It seems more portable and works on macos too.

opensuse defines `OSTYPE` as `linux`, ubuntu defines it as `linux-gnu`  and alpine (for example) doesn't define it, where as `uname` is just Linux on all of them

Reply:
👍  I'm happy with this as is - especially since thats currently how it is - but it'd be nice to see us use uname instead.